### ISSUE TITLE
arduino-builder issues

### ISSUE THREAD

**Issue by eulesv** [esp8266/esp8266-wiki#64]
Hello all,

I don't really know if this is the best place to ask, so please bear with me :) I'm trying to setup continuous deployment with some the esp8266. I decided on the arduino-builder tool on Debian, first I found some issues mostly related to the paths but I was able to overcome those, then I got some errors related to ctags (ctags pattern missing), I managed to get this to work by adding some lines from the default Arduino platform file; then I got this error "cannot open linker script file {build.flash_ld}" I tried feeding the value via --prefs but it didn't work, so I gave up on Linux (I'm a windows guy) and decided to simply make it work on my existing Windows setup.
Long story short I ended up having the exact same issues... so my questions:

1. Why platforms.txt doesn't include a ctags pattern?
2. How is it different compiling from the Arduino IDE and Arduino-builder? (my sketch is very simple and it works on the IDE)
3. What is the requirements for ld? Any pointers on how I can make this work?

Thanks all!
--Rp

**Comment by kubark42** [comment]
@eulesv Could you post the script as you have it so far?
